File: Sirma Academy May 2024/Java/Conditional Statements/src/ArchitecturalEraIdentifier.java
import java.util.Scanner;

public class ArchitecturalEraIdentifier {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int year = Integer.parseInt(sc.nextLine());
        String material = sc.nextLine();
        String output = "";

        if(year < 500 && material.equals("stone")){
            output = "Ancient";
        } else if(year >= 500 && year <= 1500 && material.equals("stone")){
            output = "Medieval";
        } else if(year >= 1500 && year <= 1800 && material.equals("wood")){
            output = "Colonial";
        } else if(year >= 1800 && year <= 1900 && material.equals("steel")){
            output = "Industrial";
        } else if(year > 1900 && material.equals("steel")){
            output = "Modern";
        } else {
            output = "Uncertain";
        }

        System.out.println(output);
    }
}

File: Sirma Academy May 2024/JavaScript/Conditional Statements/ArchitecturalEraIdentifier.js
function eraIdentifier(year, material) {
  let output = "";
  material = material.toLowerCase();

  if (year < 500 && material == "stone") {
    output = "Ancient";
  } else if (year >= 500 && year <= 1500 && material == "stone") {
    output = "Medieval";
  } else if (year >= 1500 && year <= 1800 && material == "wood") {
    output = "Colonial";
  } else if (year >= 1800 && year <= 1900 && material == "steel") {
    output = "Industrial";
  } else if (year > 1900 && material == "steel") {
    output = "Modern";
  } else {
    output = "Uncertain";
  }

  console.log(output);
}

eraIdentifier(300, "stone");
